fsharp-backend/tests/testfiles/db.tests

```diff
@@ -38,6 +43,46 @@
 
 
 
+// ------------
+// Schema
+// ------------
+[test.db.schema_v0 does not exist for ocaml OCAMLONLY] with DB X
+DB.schema_v0 X = Test.typeError_v0 "DB::schema is DEPRECATED"
+
+[test.db.schema_v0 does not exist for fsharp FSHARPONLY] with DB X
+DB.schema_v0 X = Test.typeError_v0 "DB::schema_v0 was removed from Dark"
+
+[test.db.schemaFields_v1 works for db with one field] with DB X
+DB.schemaFields_v1 X = ["x"]
+
+[test.db.schemaFields_v1 works for db with multiple fields] with DB XY
+DB.schemaFields_v1 XY = ["x", "y"]
+
+[test.db.schema_v1 works for db with one field] with DB X
+DB.schema_v1 X = { x = "Str" }
+
+[test.db.schema_v1 works for db with multiple fields] with DB SortedX
+DB.schema_v1 SortedX = { x = "Str"; sortBy = "Int" }
+
+
+
+
+// ------------
+// Setting values
+// ------------
+
+// DB.set
+[test.db.set does upsert] with DB X
+(let old = DB.set_v1 { x = "foo" } "hello" X in
```

Comment:
I've been trying hard to avoid "foo", "bar" and other things in the codebase (though there are definitely some in there now). My understanding is that it makes the code less approachable, see for example [this post](https://dev.to/kiarathedev/why-i-don-t-use-foo-bar-baz-1kf7), which while it does not exactly match our situation, gives a general idea of why to avoid them.

Reply:
Understood and agreed - I was admittedly being lazy/noncreative. Pushed a fix.